package.json

```diff
@@ -20,16 +20,16 @@
   "main": "./lib",
   "dependencies": {
     "buffer-writer": "2.0.0",
+    "connection-string": "^3.0.4",
     "packet-reader": "1.0.0",
     "pg-connection-string": "0.1.3",
+    "pg-native": "^3.0.0",
```

Comment:
pg isn’t supposed to have a dependency on pg-native, since pg-native is optional.